www/src/data/sidebars/contributing-links.yaml

```
@@ -63,6 +63,22 @@
               link: /contributing/docs-writing-process/
             - title: Docs Templates
               link: /contributing/docs-templates/
+              items:
+                - title: How to Write a Reference Guide
+                  breadcrumbTitle: Reference Guide
```

```suggestion
                  breadcrumbTitle: Reference Guides
```
I'd recommend making this plural since it includes overview pages, and we can make pluralization consistent throughout.